examples/nci/nci_rf.py

```diff
@@ -0,0 +1,108 @@
+"""
+Script that trains Sklearn multitask models on nci dataset.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import os
+import numpy as np
+import shutil
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+from deepchem.datasets import Dataset
+from deepchem.models.multitask import SingletaskToMultitask
+from deepchem import metrics
+from deepchem.metrics import Metric
+from deepchem.models.sklearn_models import SklearnModel
+from deepchem.utils.evaluate import Evaluator
+from deepchem.datasets.nci_datasets import load_nci
+
+np.random.seed(123)
+
+# Set some global variables up top
+
+reload = False
+verbosity = "high"
+
+#base_data_dir = "/scratch/users/rbharath/muv"
```

Comment:
yeah I was thinking about the best design way to handle paths when you asked me the q (reason why I only mentioned having to change paths as the only thing that had me stuck for a bit). I honestly think setting one global variable at the top script (in this case, nci_rf) that is then just passed down and further edited is the best. I remember either in the *_datasets.py scripts or one of the dnn scripts a base directory has to be specified in addition to other data directories - I ended up deleting my rf script a few times on accident because of an os.shutil.rmtree command. I think it's best to only specify one directory and have the script create all others (which it almost basically does).

We can also chat in person tomorrow!